data_model/derive/src/api.rs

```diff
@@ -10,13 +10,13 @@ pub fn process_item(mut input: syn::DeriveInput) -> TokenStream {
     }
 
     let non_transparent_item = quote! {
-        #[cfg(not(feature = "transparent_api"))]
+        #[cfg(all(not(test), not(feature = "transparent_api")))]
```

Comment:
IMO `transparent_api` affecting integration tests is not desireable, ergo this change